edk2toollib/uefi/telem/plugins/example_plugin.py

```diff
@@ -0,0 +1,37 @@
+# @file a_test_plugin.py
+# A testing plugin used to verify parsing
+#
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+import sys
+import uuid
+
+sys.path.append('../..')
+from cper_section_data import SECTION_PARSER_PLUGIN
```

Comment:
I would import edk2toollib.uefi.telem.cper_section_data 
Saves you the sys path append

Reply:
My idea was that you would be able to pick up and move the telem folder wherever and still have things work. I can think of cases where the user isn't interested in the edk2tool stuff and only wants to parse CPER records.

Reply:
sorry, late joining this PR.  I think we clarified that our usage model is not really supporting that.  pytool library is a python library you import and that is how you use it.  Anything else just gets too crazy/wild west.